src/db/SamochodDriver.java

```diff
@@ -0,0 +1,82 @@
+package db;
+
+import data_structure.Samochod;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SamochodDriver {
+    public static List<Samochod> getAll() {
+        return getFromDB("");
+    }
+
+    public static List<Samochod> getByStatus(String status){
+        String filters = String.format("Status='%s'", status);
+        return getFromDB(filters);
+    }
+
+    public static Samochod get(int VIN){
+        String filters = String.format("VIN=%d", VIN);
+        List<Samochod> results = getFromDB(filters);
+        if (results.size() != 1)
```

Comment:
Co tu się dzieje? czemu != 1?

Reply:
Bo get zwraca tylko jeden, konkretny model. Jeśli rozmiar jest inny (czyli jeśli query jest na PrimaryKey to co najwyżej 0), to takiego moedlu nie ma w bazie